src/ckmeans.js

```diff
@@ -31,7 +31,7 @@
 
 function numericSort (array) {
   return array
-    // ensure the array is changed in-place
+    // ensure the array is not changed in-place
```

Comment:
Thanks! The comment actually came from [simple-statistics](https://github.com/simple-statistics/simple-statistics/blob/master/src/numeric_sort.js)–I think your change makes sense so I'll open up a PR there as well.